tests/tests_update.yml

```diff
@@ -7,14 +7,26 @@
       vars:
         crypto_policies_policy: LEGACY
         crypto_policies_reload: false
-
   tasks:
     - name: Verify the facts are correctly set
       block:
         - meta: flush_handlers
-
-        - name: Check the current policy is FUTURE
+        - name: Check the current policy is LEGACY
           assert:
             that:
-              - "crypto_policies_active == 'LEGACY'"
-              - "crypto_policies_reboot_required == true"
+              - crypto_policies_active == 'LEGACY'
+              - crypto_policies_reboot_required == true
+  roles:
+    - role: linux-system-roles.crypto_policies
+      vars:
+        crypto_policies_policy: DEFAULT:NO-SHA1
```

Comment:
I think the NO-SHA1 is in RHEL8, but I am not sure if it is in Fedora and if it will be in next RHEL (as SHA1 is already disabled in Fedora). Do we have some more common subpolicy modude that we can use, such as `OSPP`?

Reply:
As discussed offline, the NO-SHA1 is still available in newer systems mostly because it can be applied on the LEGACY policy if somebody wants so I think this is fine. Thanks.